app/modbus.mjs

```diff
@@ -62,20 +64,18 @@ export const parseTemperature = (temperature) => {
 }
 
 export const getFlagSummary = async (modbusClient) => {
-    let result = await mutex.runExclusive(async () => modbusClient.readCoils(1, 10))
+    let result = await mutex.runExclusive(async () => modbusClient.readCoils(0, 13))
     let summary = {
-        'away': result.data[0],
-        'longAway': result.data[1],
-        'overPressure': result.data[2],
-        'maxHeating': result.data[5],
-        'maxCooling': result.data[6],
-        'manualBoost': result.data[9],
-    }
-
-    result = await mutex.runExclusive(async () => modbusClient.readCoils(12, 1))
-    summary = {
-        ...summary,
-        'summerNightCooling': result.data[0],
+        'stop': result.data[0],
```

Comment:
I've previously opted to exclude this coil since it's AFAICT impossible to get the machine into a state where this returns 1. Do you guys see any use for it? @JohanElmis @tomrosenback ?

Reply:
I don´t see any use for it no, especially if you can´t trigger this state manually, might be different for the Pro unit thou, but if we by this makes the Modbus communication more stable I don´t see an issue with making this change.

Reply:
The Modbus request can remain like it is, we just don't have to expose the `stop` value. @JohanElmis you decide, you added it :shrug:

Reply:
I will comment out the stop.
But I need to read from 0. If I start at 1 and read 12, it doesn't work at all.

Reply:
I have seen some stability issues aswell on my Pandion device, have had to reboot the Pandion unit to fix the communication issue. I tried to increase the interval of publishing, not sure yet if this has helped the issue. @Jalle19 have you experienced something similar? I was thinking about if we maybe would need to optimize the communication with the Modbus device further to avoid unnecessary reads. I can create an issue for this if needed, @Jalle19 comments?

Reply:
Strange, but whatever works 👍

Reply:
Lets see if the issue remains, if it was solved by increasing the interval I´ll change the defaults in the addon repo.

Reply:
I missed your previous comment. It has happened _once_ for me that the unit rebooted itself (I noticed it while showering and the noise from the exhaust just stopped), but haven't had to reboot it manually ever. Could be a bug in the firmware, very hard to tell.

Could be a similar issue to what @JohanElmis is experiencing, i.e. many smaller Modbus reads in rapid succession makes the whole thing shit itself. It's a tricky situation - doing too large reads usually just results in an error as well, the maximum that normally seems to work is about 10 registers in my experience :shrug: 

> Lets see if the issue remains, if it was solved by increasing the interval I´ll change the defaults in the addon repo.

Sounds good! We could also look into using different intervals for different readings. The temperature readings make no sense to update every 10 seconds, they could do with 30 or 60 second intervals, while the modes/switches you want to be pretty much instant. I guess I'll make an issue for that.